basic/stam/StamExAnswer.py

```diff
@@ -0,0 +1,51 @@
+"""Reads a file, edits it and saves it in a new location."""
+import os
+
+FIRST_NAME = "Keren"
+LAST_NAME = "Guez"
+
+
+def check_exists(input_file, output_folder):
+    """Checks if paths are valid and prints a message accordingly.
+
+    Args:
+        input_file (str): The path to the file that will be edited.
+        output_folder (str) : The path to the output folder.
+    Returns:
+        True : If both paths are valid.
+        False : If any path isn't valid.
```

Comment:
in returns you write the type of the returned value and what it represents, not its values